Makefile

```diff
@@ -190,6 +193,18 @@ ifeq ($(HYPERVISOR),qemu)
     DEFNETWORKMODEL := macvtap
 endif
 
+ifeq ($(HYPERVISOR),firecracker)
+    ifeq ($(FCCMD),)
+        $(error "ERROR: $(HYPERVISOR) not available for architecture $(ARCH)")
+    endif
+
+    VALID_HYPERVISOR = true
+
+    CONFIG_FILE = configuration-fc.toml
+    DEFBLOCKSTORAGEDRIVER := virtio-mmio
```

Comment:
@jodh-intel we also need to default to vsock for firecracker

Reply:
That's already done - there isn't a build variable for vsock - it's simply enabled in the `configuration-fc.toml.in` file.